src/sgame/sg_entities.cpp

```diff
@@ -527,7 +527,6 @@ static const entityCallEventDescription_t gentityEventDescriptions[] =
 		{ "onFree",      ON_FREE      },
 		{ "onReach",     ON_REACH     },
 		{ "onReset",     ON_RESET     },
-		{ "onSpawn",     ON_SPAWN     },
```

Comment:
Do you think this is functionality we might want to keep for mods/Lua?

Reply:
The code relating to this is all specific to BSP entities so I doubt it could be good with a Lua integration.